sql/src/test/java/io/crate/expression/symbol/FunctionTest.java

```diff
@@ -72,4 +78,28 @@ public void testSerialization() throws Exception {
         }
         return features;
     }
+
+    @Test
+    public void testNullableSerializationOfNull() throws IOException {
+        BytesStreamOutput output = new BytesStreamOutput();
+        Symbols.nullableToStream(null, output);
+
+        StreamInput input = output.bytes().streamInput();
+        Symbol symbol = Symbols.nullableFromStream(input);
+
+        assertThat(symbol, is(nullValue()));
+    }
+
+    @Test
+    public void testNullableSerializationOfFunction() throws IOException {
```

Comment:
Afaik this test is not needed and won't add any coverage/value. The implementation do not differ between generic symbols or functions.